instructions/Ubuntu.rst

```diff
@@ -40,7 +40,7 @@ and then install package with: ::
 
 This will install Mantid into ``/opt/mantidnightly``. It does **not** update the environment so you must type the following to start it: ::
 
-    ./opt/mantidnightly/bin/MantidPlot
+    /opt/mantidnightly/bin/launch_mantidplot.sh
```

Comment:
Thanks for spotting this. The `bin` directory should contain a `MantidPlot` symbolic link to `launch_mantidplot.sh` - the one I've just installed has it. Is it just the erroneous `.` as the first character that is the problem here? If so could we leave `MantidPlot` as the command to run?

Reply:
Ah, I assumed that was the binary. You're right, removing the "." works fine"